src/docs/JobPlanner/references/JobPlannerServiceReference.adoc

```diff
@@ -204,3 +204,74 @@ Given an event, a predefined action will be applied on the workflow execution.
 If an inclusion or exclusion calendar is not retrievable, it is blocking the <<_glossary_workflow,*Workflow*>> submission.
 An inclusion or exclusion calendar can become not retrievable if it cannot be downloaded from its URL and the
 <<_glossary_job_planner,*Job Planner*>> cache doesn't hold a copy.
+
+== Notifications
+
+The <<_glossary_job_planner,*Job Planner*>> can send notifications to users when particular events occur. The <<_glossary_notification_service,*Notification service*>> documentation can be found link:../admin/ProActiveAdminGuide.html#_notification_service[here].
```

Comment:
Shouldn’t you put to subscribed users? (I'm not sure but i remember that you need to subscribe to the notification and activate it in order to receive the notifications)

Reply:
I just noticed that describe how in the following paragraphs. But you can put here if you want "subscribed users"

Reply:
I think that in that case I have to change the sentence to: "The Job planner sends notifications to subscribed users ...".

If you don't mind I'll leave it as it is.

Reply:
You can leave it as it is. I don't mind it.

Reply:
Ok i'll do that. Thanks.